lib/logger_json/formatters/elastic.ex

```diff
@@ -95,6 +96,37 @@ defmodule LoggerJSON.Formatters.Elastic do
     "message" => "oops!"
   }
   ```
+
+  You can also choose to log caught exceptions with a custom message.
+  For example, after catching an exception with `try`/`rescue`:
+
+  ```elixir
+  try do
+    raise "oops"
+  rescue
+    e in RuntimeError -> Logger.error("Something went wrong", crash_reason: e)
```

Comment:
I think we can just document it to be a tuple that all providers support, so we won't need to add any special handling for this formatter:
```suggestion
    e in RuntimeError -> Logger.error("Something went wrong", crash_reason: {e, __STACKTRACE__)
```

Reply:
I didn't know it was possible to use crash_reason like that! Thanks!

I was trying `Logger.error("Something went wrong", crash_reason: e)` with the implementation of the Elastic logger before this MR's changes and noticed that the test would just hang indefinitely.

I've removed my special code handling for it and cemented this crash_reason form in a test for the Elastic logger. I also left it in the documentation as it's quite useful info to know as someone implementing logging :)